### ISSUE TITLE
.ahk files not working

### ISSUE THREAD

**Issue by kimberley1111** [TaranVH/2nd-keyboard#162]
Ok, I install LuaMacro and keypressed.txt shows the correct button I pressed and no key of the second keyboard is working anymore, so that works...

I also download the 'Photoshop_Functions.ahk' file and Run Script, but nothing actually works. Whatever I press on the second keyboard, nothing seems to happen. I see the little icon in the tray. Not in Photoshop or a new Notepad. 

Do I need to write my own shortcuts? Or what does the .ahk file does?

**Comment by litsher** [comment]
Hey hey, 

The AHK files refer to autohotkey! 
You might need to download that to make it work.

Hope it helps! :)

**Comment by kimberley1111** [comment]
Nope, I already did that! But now I have another .ahk file that works! 

I think this file 'https://github.com/TaranVH/2nd-keyboard/blob/master/Photoshop_Functions.ahk' is corrupted. It doesn't do anything...

**Comment by AadiSahni** [comment]
You have the functions ahk which includes just the actions, not keymappings. You need to copy paste keymappings from All_multiple_keyboard_assignments.ahk into your own ahk script with the same name. If you're still confused, watch the linked video in the readme.md
